test/Interop/Cxx/reference/reference.swift

```diff
@@ -103,4 +103,9 @@ ReferenceTestSuite.test("const reference to template") {
   expectEqual(53, ref.pointee)
 }
 
+ReferenceTestSuite.test("rvalue reference of trivial type") {
+  setStaticIntRvalueRef(consuming: 2)
+  expectEqual(2, getStaticInt())
```

Comment:
Just curious, how would this test have failed prior to your change?

Reply:
It crashed with bad memory access. We passed 2 directly as if it was the address of the value.